README.md

```diff
@@ -29,24 +29,32 @@ export AUTH_FILE=<path to service-account.json>
     -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -f manifests/cluster.yaml
 ```
 
-### Deploy Prometheus-Meta & Grafana
-> This is used for collecting and displaying the test results.
+### Deploy monitoring components
+> Collecting, monitoring and displaying the test results
 
 ---
 
+- [Optional] If used with the Github integration generate a GitHub auth token.
+  * Login with the [Prombot account](https://github.com/prombot) and generate a [new auth token](https://github.com/settings/tokens).
+  * With permissions: `public_repo`, `read:org`, `write:discussion`.
 - Set the following environment variables
 ```
 export GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL=<client-email present in service-account.json>
 export GRAFANA_ADMIN_PASSWORD=password
 export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom domain.
```

Comment:
export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom domain or an empty string when not used with the Github integration.